core/src/main/com/github/mmorihiro/core/MakeNewGeneration.kt

```diff
@@ -3,6 +3,48 @@ package com.github.mmorihiro.core
 
 internal class MakeNewGeneration {
     fun get(array: Array<BooleanArray>): Array<BooleanArray> {
-        return array
+        val copy = copy(array)
+        for (i in array.indices) {
+            for (j in array[0].indices) {
```

Comment:
May I fix [MakeNewGeneration.kt#L28](https://github.com/mmorihiro/LifeGame/blob/78a8da16aa67698d564fe676432eeea151689e13/core/src/main/com/github/mmorihiro/core/MakeNewGeneration.kt#L28) and [MakeNewGeneration.kt#L42](https://github.com/mmorihiro/LifeGame/blob/78a8da16aa67698d564fe676432eeea151689e13/core/src/main/com/github/mmorihiro/core/MakeNewGeneration.kt#L42), too?